README.md

```diff
@@ -1,60 +1,28 @@
 [![Coverage Status](https://img.shields.io/codecov/c/github/auth0/java-jwt.svg?style=flat-square)](https://codecov.io/github/auth0/java-jwt)
-[![License](https://img.shields.io/:license-mit-blue.svg?style=flat)](https://doge.mit-license.org)
-[![Javadoc](https://javadoc.io/badge2/com.auth0/java-jwt/javadoc.svg)](https://javadoc.io/doc/com.auth0/java-jwt/latest/index.html)
-
-A Java implementation of [JSON Web Token (JWT) - RFC 7519](https://tools.ietf.org/html/rfc7519).
-
-> :warning:  **Important security note:** JVM has a critical vulnerability for ECDSA Algorithms - [CVE-2022-21449](https://nvd.nist.gov/vuln/detail/CVE-2022-21449). Please review the details of the vulnerability and update your environment.
+[![License](http://img.shields.io/:license-mit-blue.svg?style=flat)](https://doge.mit-license.org/)
+[![Maven Central](https://img.shields.io/maven-central/v/com.auth0/java-jwt.svg?style=flat-square)](https://mvnrepository.com/artifact/com.auth0/java-jwt)
+[![javadoc](https://javadoc.io/badge2/com.auth0/auth0/javadoc.svg)](https://javadoc.io/doc/com.auth0/java-jwt)
 
-If you're looking for an **Android** version of the JWT Decoder take a look at our [JWTDecode.Android](https://github.com/auth0/JWTDecode.Android) library.
+:books: [Documentation](#documentation) - :rocket: [Getting Started](#getting-started) - :computer: [API Reference](#api-reference) :speech_balloon: [Feedback](#feedback)
 
-> You are viewing the documentation for the v4 beta release. For the latest stable release, please see the [version 3.x documentation](https://github.com/auth0/java-jwt).
+## Documentation
+- [Examples](./EXAMPLES.md) - code samples for common auth0-java scenarios.
+- [Docs site](https://www.auth0.com/docs) - explore our docs site and learn more about Auth0.
 
-## Table of Contents
-- [**Requirements**](#requirements)
-- [**Installation**](#installation)
-- [**Available Algorithms**](#available-algorithms)
-- [**Quickstart**](#quickstart)
-  + [**Create and Sign a Token**](#create-and-sign-a-token)
-  + [**Verify a Token**](#verify-a-token)
-- [**Usage**](#usage)
-  + [**Pick the algorithm**](#pick-the-algorithm)
-  + [**Time Validation**](#time-validation)
-  + [**Header Claims**](#header-claims)
-  + [**Payload Claims**](#payload-claims)
-  + [**Claim Class**](#claim-class)
-- [**Javadoc**](https://javadoc.io/doc/com.auth0/java-jwt/latest/index.html)
+## Getting Started
 
-## Requirements
+### Requirements
 
-This library is supported for Java 8, 11, and 17. For issues on non-LTS versions above 8, consideration will be given on a case-by-case basis.
+This library is supported for Java LTS versions 8, 11, and 17. For issues on non-LTS versions above 8, consideration will be given on a case-by-case basis.
 
-## Installation
+> `auth0-java` is intended for server-side JVM applications. Android applications should use the [Auth0.Android SDK](https://github.com/auth0/auth0.android).
```

Comment:
Is this reference to `auth0-java` intentional?

Reply:
No, should be java-jwt. Pushed new commit to fix.